core/src/main/java/org/opensearch/sql/data/model/ExprDateValue.java

```diff
@@ -67,7 +67,7 @@ public LocalDateTime datetimeValue() {
 
   @Override
   public Instant timestampValue() {
-    return ZonedDateTime.of(date, timeValue(), ZoneId.systemDefault()).toInstant();
+    return ZonedDateTime.of(date, timeValue(), ZoneId.of("UTC")).toInstant();
```

Comment:
If the intent is to use the same timezone as `ExprTimestampValue`, it'd be good to use `ExprTimestampValue.ZONE` here to make the intent clear.

Reply:
Fixed in 8cd8f783, thanks.